### ISSUE TITLE
Add top menu item for calendar

### ISSUE THREAD

**Issue by gvwilson** [rstudio/education.rstudio.com#62]
I always have to hunt around to find the calendar of upcoming events (I never remember to look for it under "blog"). Could we please add an item to the menu at the top?

**Comment by apreshill** [comment]
Have you noticed the calendar icon in the footer?
![Screen Shot 2019-10-03 at 4 01 24 PM](https://user-images.githubusercontent.com/12160301/66102869-2875c380-e5f7-11e9-91a4-b8cf846d632a.png)

Is that enough to solve the problem? I really like the simplicity of the 4 menu items up top.

**Comment by gvwilson** [comment]
I agree about the simplicity of the top menu, but I've never noticed that icon before... Maybe it's just me?

**Comment by apreshill** [comment]
Having thought about this a bit more, I'd like to keep the top menu the same but expand our footer to be less cryptic and have words like 

Events
Team
Community

with links out. Would this be an acceptable solution for you?

**Comment by gvwilson** [comment]
Sure - if nobody else was as unobservant as I was it's probably not worth worrying about.

**Comment by apreshill** [comment]
it was one of those things that bothered me but I decided to let it slide until it bothered someone else ;) So worth fixing at some point, we'll leave this open and see when I figure out a CSS solution (http://tachyons.io/components/#footers)

**Comment by cdhowe** [comment]
Did you add the calendar to the top menu? I'm not seeing it on that link.

**Comment by apreshill** [comment]
@cdhowe no I made the footer more "chatty"- if you scroll to the bottom you can see words now in the footer instead of icons. I liked having the four top menu items as they are, but I'm open to adding events to the top menu bar instead.

**Comment by cdhowe** [comment]
Sorry -- I was just going by what was in the subject line.....

**Comment by apreshill** [comment]
Note to self, I like this footer: https://github.com/rstudio/rstudio.com/blob/master/layouts/partials/footer-skinny.html
